### PR TITLE
18761, 18763 - EFT statement settings updates

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:*
https://github.com/bcgov/entity/issues/18761
https://github.com/bcgov/entity/issues/18763

*Description of changes:*
- Fix issue where statement settings are always hidden when the feature flag for eft payment method is enabled. Statements settings are now always shown, but the statement frequency will be locked if the feature flag is enabled and payment method is EFT
- Fix some styling issues as per UXPIN

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by ochiu** [comment]
Note, a small pay-api update is coming in another PR for this ticket is coming to automatically enable notifications and populate admin as recipient if there are no current recipients.

**Review by seeker25 — COMMENTED** [review]
see above

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2659-nc5eu9he.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2659-nc5eu9he.web.app

File: auth-web/src/components/auth/account-settings/statement/StatementsSettings.vue [hunk 1]
```diff
@@ -200,10 +219,12 @@
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
-import { Member, MembershipType, Organization } from '@/models/Organization'
+import { LDFlags, PaymentTypes } from '@/util/constants'
+import { Member, MembershipType, OrgPaymentDetails, Organization } from '@/models/Organization'
 import { StatementListItem, StatementNotificationSettings, StatementRecipient, StatementSettings } from '@/models/statement'
 import { mapActions, mapState } from 'pinia'
 import CommonUtils from '@/util/common-util'
+import LaunchDarklyService from 'sbc-common-components/src/services/launchdarkly.services'
```
[the remark this answers is not in the document]
Comment (ochiu) [reply auth-web/src/components/auth/account-settings/statement/StatementsSettings.vue:226]: Updated

[the remark this answers is not in the document]
Comment (seeker25) [reply auth-web/src/components/auth/account-settings/statement/StatementsSettings.vue:226]: Looks good, might be worth adding another point to the tickets (extra work done here)

[the remark this answers is not in the document]
Comment (ochiu) [reply auth-web/src/components/auth/account-settings/statement/StatementsSettings.vue:226]: bumped #18763 by 1pt

**Review by seeker25 — APPROVED** [review]

**Pull request merged**